Gems/AtomLyIntegration/EMotionFXAtom/Code/Tools/EMStudio/AnimViewportRenderer.cpp

```diff
@@ -122,22 +120,15 @@ namespace EMStudio
         const AZ::Render::LightingPreset* preset = lightingPresetAsset->GetDataAs<AZ::Render::LightingPreset>();
         SetLightingPreset(preset);
 
-        // Create grid
+        // Create the ground plane
         AzFramework::EntityContextRequestBus::EventResult(
-            m_gridEntity, entityContextId, &AzFramework::EntityContextRequestBus::Events::CreateEntity, "ViewportGrid");
-        AZ_Assert(m_gridEntity != nullptr, "Failed to create grid entity.");
+            m_groundEntity, entityContextId, &AzFramework::EntityContextRequestBus::Events::CreateEntity, "ViewportModel");
+        AZ_Assert(m_groundEntity != nullptr, "Failed to create model entity.");
 
-        AZ::Render::GridComponentConfig gridConfig;
-        gridConfig.m_secondarySpacing = m_renderOptions->GetGridUnitSize();
-        gridConfig.m_axisColor = m_renderOptions->GetMainAxisColor();
-        gridConfig.m_primaryColor = m_renderOptions->GetGridColor();
-        gridConfig.m_secondaryColor = m_renderOptions->GetSubStepColor();
-        auto gridComponent = m_gridEntity->CreateComponent(AZ::Render::GridComponentTypeId);
-        gridComponent->SetConfiguration(gridConfig);
-
-        m_gridEntity->CreateComponent(azrtti_typeid<AzFramework::TransformComponent>());
-        m_gridEntity->Init();
-        m_gridEntity->Activate();
+        m_groundEntity->CreateComponent(AZ::Render::MeshComponentTypeId);
+        m_groundEntity->CreateComponent(AZ::Render::MaterialComponentTypeId);
+        m_groundEntity->CreateComponent(azrtti_typeid<AzFramework::TransformComponent>());
+        m_groundEntity->Activate();
```

Comment:
Shall we move this to a separate function? Let's also keep the previous version with the grid as that should be an option later on so users can decide if they prefer the grid of the solid checkerboard ground plane.